### ISSUE TITLE
The Login warning message on the sign in page doesn't have enough contrast  

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/scigateway#791]
**Description**:
## Light Mode

The warning message when an incorrect password and username doesn't have enough contrast to satistfy WCAG 2 AA. Suggested foreground color change to #e50000 

![login(lightmode)](https://user-images.githubusercontent.com/83226114/139666993-53345c2b-9ad4-4eb4-ae22-77779daa6ee2.png)

<br><br>
## Dark Mode

The warning message when an incorrect password and username doesn't have enough contrast to satistfy WCAG 2 AA.

![login(darkmode)](https://user-images.githubusercontent.com/83226114/139666981-0276328e-3d4a-4edf-9814-7e8f9ad1ddb8.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done
